features/admin.organizations.v1/components/add-organization-modal.tsx

```diff
@@ -387,18 +424,22 @@ export const AddOrganizationModal: FunctionComponent<AddOrganizationModalPropsIn
      * @returns Organization handle unique validation.
      */
     const renderOrgHandleUniqueValidationIcon = (): ReactElement => {
+
         if (!orgHandle || orgHandle === OrganizationManagementConstants.ORG_HANDLE_PLACEHOLDER) {
             return <Icon name="circle" color="grey" />;
         }
 
-
-        if (orgHandle && !isCheckingOrgHandleValidity && !isOrgHandleDuplicate) {
+        if (isCheckingOrgHandleValidity) {
+            return <Icon name="circle" color="grey" />;
+        }
+        if (isOrgHandleFieldFocused) {
+            return <Icon name="circle" color="grey" />;
+        }
+        if (orgHandle && !isOrgHandleDuplicate) {
```

Comment:
how about combining the logic for displaying the `<Icon name="circle" color="grey" />` into one if statement for better readability?